edx_solutions_api_integration/users/views.py

```diff
@@ -391,10 +391,9 @@ def delete(self, request):
         """
         # require at least either ids or username filters
         if set(self.request.query_params.keys()) & set(['username', 'ids']):
-            qs = self.filter_queryset(self.queryset)
-            for user in qs:
-                CCUser.from_django_user(user).retire('Deleted username')
```

Comment:
The comments are retired here https://github.com/edx-solutions/edx-platform/pull/1359/files#diff-df4681b6dbe6bbc209ddf0f521adb0a6R564